katas/6kyu/counting-duplicates.js

```diff
@@ -0,0 +1,7 @@
+function duplicateCount(text) {
+    const letters = text.toLowerCase().split('');
+    const dubbles = letters.filter((letter, index, array) => array.indexOf(letter) !== index);
```

Comment:
as I said, try to avoid oneline code solutions and write more human-readable code. don't scare to write more code but you have to completely understand what it does even in 1 week, month, year. it has to be clear for you and your teammates